specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -245,7 +250,8 @@
             "$ref": "./examples/ContainerLogsList.json"
           }
         },
-        "description": "Get the logs for this container.",
+        "summary": "Get the logs for a specified container instance in a specified resource group and container group.",
```

Comment:
You may want to address this warning: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/270502075#L673
"The summary and description values should not be same."